lib/puppet/provider/mongodb_user/mongodb.rb

```diff
@@ -133,7 +92,7 @@ def password_hash=(_value)
   end
 
   def password=(value)
-    if mongo_24? || mongo_26?
+    if mongo_26?
```

Comment:
The docs indicate this method should also work on 3.x and 4.x but I'll look into that later when I have acceptance tests